app/graphql/mutations/questions/create.rb

```diff
@@ -0,0 +1,22 @@
+module Mutations
+  module Questions
+    class Create < Mutations::BaseMutation
+      graphql_name 'CreateQuestion'
+
+      argument :survey_id, ID, required: true
+      argument :title, String, required: true
+      argument :tag, String, required: false
+
+      type Types::CreateRatingQuestionResult
+
+      def resolve(**args)
```

Comment:
Use keyword arguments here:

```suggestion
      def resolve(survey_id:, title:, tag:)
```

Reply:
I chose `**args` because `tag` is optional.